DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.LearningMenu;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class SectionContentViewModelTests
+    {
+        [Test]
+        public void Section_content_should_have_title()
+        {
+            // Given
+            const string applicationName = "Application name";
+            const string customisationName = "Customisation name";
+            const int customisationId = 5;
+            var sectionContent = SectionContentHelper.CreateDefaultSectionContent(
+                applicationName: applicationName,
+                customisationName: customisationName
+            );
+
+            // When
+            var sectionContentViewModel = new SectionContentViewModel(sectionContent, customisationId);
+
+            // Then
+            sectionContentViewModel.CourseTitle.Should().Be($"{applicationName} - {customisationName}");
+        }
+
+        [Test]
+        public void Section_content_should_have_section_name()
+        {
+            // Given
+            const string sectionName = "Section name";
+            const int customisationId = 5;
+            var sectionContent = SectionContentHelper.CreateDefaultSectionContent(
+                sectionName: sectionName
+            );
+
+            // When
+            var sectionContentViewModel = new SectionContentViewModel(sectionContent, customisationId);
+
+            // Then
+            sectionContentViewModel.SectionName.Should().Be(sectionName);
+        }
+
+        [Test]
+        public void Section_content_with_has_learning_should_have_percent_complete()
+        {
+            // Given
+            const bool hasLearning = true;
+            const double percentComplete = 0.5;
+            const int customisationId = 5;
+            var sectionContent = SectionContentHelper.CreateDefaultSectionContent(
+                hasLearning: hasLearning,
+                percentComplete: percentComplete
+            );
+
+            // When
+            var sectionContentViewModel = new SectionContentViewModel(sectionContent, customisationId);
+
+            // Then
+            sectionContentViewModel.PercentComplete.Should().Be($"{percentComplete:f0}% Complete");
```

Comment:
I'd prefer for this to be written out explicitly instead of relying on `f0`. It makes the test a bit easier to read and means we're not relying on logic which could be wrong 🙂